scripts/sct_dmri_eddy_correct.py

```diff
@@ -14,19 +14,20 @@
 # TODO: add test.
 # TODO: remove FSL dependency
 
-# check if needed Python libraries are already installed or not
-import sys
-import os
 import commands
 import getopt
+import os
+import shutil
+import sys
 import time
+
 import numpy as np
-from msct_image import Image
+
 import sct_utils as sct
-# get path of the toolbox
-status, path_sct = commands.getstatusoutput('echo $SCT_DIR')
+from msct_image import Image
 
 
+status, path_sct = commands.getstatusoutput('echo $SCT_DIR')
```

Comment:
status is never used, and it should be  path_sct = os.getenv('SCT_DIR'). echo would also crash  in windows.